lib/ffmpegCommandService.js

```diff
@@ -364,9 +364,18 @@ class FfmpegCommandService {
             });
         }
 
-        // 3a) Trim videos
+        // 3a) Trim, Scale videos
+        // NOTE: If we don't scale as part of the concat then we end up with issues for videos of different sizes
+        const outWidth = command.output.dimensions.width;
+        const outHeight = command.output.dimensions.height;
+        const ratio = `${outWidth}/${outHeight}`;
+
         command.video.forEach((video,i) => {
-            filter += ` [${i}:v] trim=start=${video.trimStart || 0}:duration=${video.trimDuration || 600}, setpts=PTS-STARTPTS [v${i}];`
+            filter += ` [${i}:v]`; // start
+            filter += ` pad=ih*${ratio}/sar:ih:(ow-iw)/2:(oh-ih)/2,`; // Force padding for videos not in the correct aspect ratio
+            filter += ` scale=${outWidth}:${outHeight}:force_original_aspect_ratio=decrease, setsar=1,`; // Scale videos to the same size (setsar=1 recommended)
```

Comment:
We have to scale each video vs scaling the end, otherwise fonts end up unpredictable and it makes it impossible to stitch videos of different sizes together.